tbx/core/blocks.py

```diff
@@ -567,6 +567,10 @@ def clean(self, value):
         errors = defaultdict(ErrorList)
         non_block_errors = ErrorList()
 
+        if value["intro"] and not value["title"]:
+            message = "You cannot add an intro without also adding a title"
+            non_block_errors.append(ValidationError(message))
```

Comment:
Please see my comment on the PhotoCollage (which I looked at first :slightly_smiling_face: )

```suggestion
            errors["title"].append(ValidationError(message))
```

which results in 

<details>
  <summary>Screenshot</summary>
  
![Screenshot from 2024-04-17 19-21-40](https://github.com/torchbox/torchbox.com/assets/7713776/0ae19849-a9eb-445f-bc49-dc4da57a06d9)
  
</details>